MIGRATION.md

```diff
@@ -43,7 +43,33 @@ In version 8 of this library, the following components have been added or remove
 ### Removed
 
 - Typography
-    - This component has been removed from the library can be replaced by [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper.
+    - These components have been removed from the library and can be replaced by the [Text](https://callstack.github.io/react-native-paper/docs/components/Text/) component from React Native Paper.
+    - The `Text` component does not support the `color` prop. It must be styled through the `style` prop and passing the value from the `theme` object returned by [useExtendedTheme()](https://github.com/etn-ccis/blui-react-native-themes?tab=readme-ov-file#typescript).
+    - Please follow the table below for the mapping of Typography and its replacement:
+
+<div style="overflow: auto">
+
+| Typography            | Replacement                                                           | 
+| --------------------- | --------------------------------------------------------------------- | 
+| \<H1>Your Text\</H1>    | <Text variant={'displayLarge'} style={{fontSize: 96, letterSpacing: 0}}>Your Text\</Text>| 
+| \<H2>Your Text\</H2>    | <Text variant={'displayMedium'} style={{fontSize: 60}}>Your Text\</Text>| 
+| \<H3>Your Text\</H3>    | <Text variant={'displaySmall'} style={{fontSize: 48}}>Your Text\</Text>| 
+| \<H4>Your Text\</H4>    | <Text variant={'headlineLarge'} style={{fontSize: 34}}>Your Text\</Text>| 
+| \<H5>Your Text\</H5>    | <Text variant={'headlineMedium'} style={{fontSize: 24}}>Your Text\</Text>| 
+| \<H6>Your Text\</H6>    | <Text variant={'headlineSmall'} style={{fontSize: 20, fontWeight: 600}}>Your Text\</Text>| 
+| \<Body1>Your Text\</Body1>    | <Text variant={'bodyLarge'} style={{letterSpacing: 0}}>Your Text\</Text>| 
+| \<Body2>Your Text\</Body2>    | <Text variant={'bodyMedium'}>Your Text\</Text>| 
+| \<Label>Your Text\</Label>    | <Text variant={'bodyLarge'} style={{letterSpacing: 0}}>Your Text\</Text>| 
+| \<Subtitle1>Your Text\</Subtitle1>    | <Text variant={'titleMedium'} style={{letterSpacing: 0}}>Your Text\</Text>|
+| \<Subtitle2>Your Text\</Subtitle2>    | <Text variant={'titleSmall'} style={{letterSpacing: 0}}>Your Text\</Text>| 
+| \<Button>Your Text\</Button>    | <Text variant={'labelLarge'} style={{letterSpacing: 0}}>Your Text\</Text>| 
+| \<Caption>Your Text\</Caption>    | <Text variant={'bodySmall'}>Your Text\</Text>| 
+| \<Overline>Your Text\</Overline>    | [\<Overline>Your Text\</Overline>](./docs/Overline.md)| 
+
+</div>
+
+> **NOTE**: The typography structure in Material Design 2 and Material Design 3 will not be a one-to-one mapping. It is better to reach out to a UI designer (especially for heading texts) if your project will be completely designed and developed in Material Design 3.
```

Comment:
```suggestion
> **NOTE**: If you are planning to fully embrace Material 3 design standards, it is better to reach out to a UI designer (especially for heading texts) in order to select the most appropriate new type styles for your application instead of mapping the old Material 2 styles into the new system. 
```